haystack/pipelines/standard_pipelines.py

```diff
@@ -231,6 +231,8 @@ def print_eval_report(
             "document_id_or_answer",
         ] = "document_id_or_answer",
         answer_scope: Literal["any", "context", "document_id", "document_id_and_context"] = "any",
+        wrong_examples_fields: List[str] = ["answer", "context", "document_id"],
+        max_characters_per_field: int = None,
```

Comment:
I tried it out. Looks good but I think `150` would be a good default value here. How I came up with that? `FARMReader` has already a `context_size` param which cuts the context to show. This value is 50, but words not chars. 150 seems to be roughly the equivalent of that. Also 150 chars fit nicely in one line on the screen:

Sample output from Tutorial 5:
```
Documents: 
 	context: people considered righteous before God. God has such a book, and to be blotted out of it signifies death. It is with reference to the Book of Life tha
 	document_id: 737cb9cd71126bd25296bce035a5956f-1

 	context: as adversaries (of God). Also, according to ib. xxxvi. 10, one who contrives evil against his neighbor will be blotted out of the Book of Remembrance 
 	document_id: fcfe96ddea6611b56ffaeda010d3a111-2

 	context: the citizens' registers. The life which the righteous participate in is to be understood in a temporal sense. In Dan. xii. 1, however, those who are f
 	document_id: 5f3f7a6be74e24b1065a4d770585d210-6

 	context: apostles' names are ``written in heaven'' (Luke x. 20), or ``the fellow-workers'' of Paul (Phil. iv. 3), and ``the assembly of the first-born'' (Heb. 
 	document_id: bcc533ac40605a5faa63f4b0d7124404-3

 	context: The Absolutely True Diary of a Part-Time Indian - wikipedia The Absolutely True Diary of a Part-Time Indian Jump to: navigation, search The Absolutely
 	document_id: edba8a30690fe62dd0330344f6ad98ba-0
```
```
Answers: 
 	answer: her cousin
 	context: ng a story in the newspaper of a 12-year-old girl getting pregnant by her cousin and trying to dispose of the baby in a trash can. Parts of the video 
 	document_id: a38e3ec379fea7e5e4ad9a620be112c5-3

 	answer: nan
 	context: nan
 	document_id: nan

 	answer: her cousin
 	context: a newspaper article about a 12-year-old girl who became pregnant from her cousin and threw the baby into a trash compactor. Contents (hide) 1 Lyrics 2
 	document_id: 4901594d2eed9d96c0ab0902829ff7fd-1

 	answer: Dave Hollister
 	context: is debut album, 2Pacalypse Now. The song, which features R&B singer Dave Hollister, is about a 12-year-old girl named Brenda who lives in a ghetto and
 	document_id: cdf5968ebecb70e11c4812fad547e929-0
```